.github/workflows/go.yml

```
@@ -0,0 +1,23 @@
+name: Go
+
+on: [push]
```

```suggestion
on:
  - push
```

[Reference](https://github.com/kohofinancial/documentation/blob/main/best%20practices/github-actions.md#cosmetics)